apps/condo/domains/user/components/auth/InputPhoneForm.tsx

```diff
@@ -149,7 +143,7 @@ export const InputPhoneForm: React.FC<IInputPhoneFormProps> = ({ onFinish }) =>
                 >
                     <Row style={ROW_STYLES} gutter={[0, 28]}>
                         <ResponsiveCol span={24}>
-                            <Row>
+                            <Row gutter={[0, 28]}>
```

Comment:
consider moving static arrays outside of components, since they're reallocating on each serenader when passed inline like that